myftp.py

```diff
@@ -9,14 +9,19 @@ def initConnection():
   hostName = sys.argv[1]
   if (hostName == 'test'):
     hostName = ftputils.TEST_HOST
-  ftpController = FTPController(hostName)
-  print(ftpController.connect())
-  return ftpController
+    ftpController = FTPController(hostName)
```

Comment:
What if the user inputs their own FTP server host? Perhaps we can just use a `sys.exit()` [here](https://github.com/Bruception/CNT4713-Project1/blob/master/myftp.py#L6).